### PR TITLE
Add Jellyfin for Android TV 0.18 release blog post

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Note: I'll merge this myself once the release is made.

### CODE REVIEW

File: blog/2024/11-30-android-tv-18/index.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+To make these changes happen the preferences for subtitles have also slightly changed. It is now possible to choose your own background and
+outline colors.
+
+### Sing along with lyrics
+
+Not only did video playback improve, playing music has gotten some upgrades as well! The most exciting change here is support for lyrics.
+Whenever your song has lyrics available (added by a plugin or on your file system), they will be shown in the app. This works especially
+well when combined with our screensaver, which will show lyrics fullscreen.
+
+**TODO SCREENSHOT**
```
Comment (JPVenson) [inline blog/2024/11-30-android-tv-18/index.mdx:60]: Todo or NotToDo

Comment (nielsvanvelzen) [reply blog/2024/11-30-android-tv-18/index.mdx:60]: I don't really have content I can use for the screenshot on my server, need  to figure out something for that 👀.

File: blog/2024/11-30-android-tv-18/index.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+- **Ask to skip**
+
+  Shows a popup that the segment can be skipped by pressing the OK button on your remote and disappears after 8 seconds or when pressing the
+  back button.
+
+  ![Ask to skip button shown during video playback](./asktoskip.png)
+
+More actions and functionality related to media segments is already in the works for future releases, like a "mute" action that temporarily
+silences audio.
```
[the remark this answers is not in the document]
Comment (nielsvanvelzen) [reply blog/2024/11-30-android-tv-18/index.mdx:37]: I did have this in one of my earlier drafts but I ended up not linking to the documentation page as I didn't see much benefit to it. The page is very barebones and doesn't have that much additional information compared to the description in the blog post.

File: blog/2024/11-30-android-tv-18/index.mdx (hunk 1 above)
Comment (joshuaboniface) [reply blog/2024/11-30-android-tv-18/index.mdx:60]: Demo server?

**Review by joshuaboniface — APPROVED** [review]

File: blog/2024/11-30-android-tv-18/index.mdx (hunk 1 above)
Comment (nielsvanvelzen) [reply blog/2024/11-30-android-tv-18/index.mdx:60]: Unfortunately we don't have any lyrics on the demo server

File: blog/2024/11-30-android-tv-18/index.mdx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+[GitHub](https://github.com/jellyfin/jellyfin-androidtv/releases/tag/v0.18.0). I've listed a subset of all changes below.
+
+### Skipping intros, outros and more with media segments
+
+Media segments are a new feature added in Jellyfin 10.10. A segment provides metadata for specific parts of a video or audio file. This can
+be used to mark an intro in your episode or the credits of a movie. The Android TV app can now use those segments to skip parts. By default,
+the app will ask to skip intros and outros, but you can fine-tune this to your liking within the app preferences. All media segment types
+are supported: intro, outro, preview, recap and commercial with two actions to choose from:
+
+- **Skip**
```
Comment (1hitsong) [inline blog/2024/11-30-android-tv-18/index.mdx:25]: Is none or do nothing an option like on web?

Comment (1hitsong) [reply blog/2024/11-30-android-tv-18/index.mdx:25]: I do. Only mentioning there being 2 options makes it sound like you must have it either auto-skip or display the button. But you _can_ choose to simply ignore the segment if you like.

Comment (nielsvanvelzen) [reply blog/2024/11-30-android-tv-18/index.mdx]: Added

> - **Do nothing**  
>   The segment will be ignored, with no skips or popups.

**Review by 1hitsong — APPROVED** [review]

**Review by JPVenson — APPROVED** [review]

**Pull request merged**